Fix incorrect grammar (#5626)
prettier/prettier: docs/rationale.md

});
```

> Prettier special cases work with common test framework functions such as `describe`, `it` and `test`.

Reviewer: ```suggestion
Prettier has special cases for common testing framework functions such as `describe`, `it` and `test`.
```
